### PR TITLE
tried to solve  unbounded ripple effect problem 

### PR DESCRIPTION
**Pull request by PremChavhan**
tried to solve the issue of unbounded ripple in navigationdrawer

### CODE REVIEW

**Comment by PatilShreyas** [comment]
Thank you @PremChavhan. Can you attack Screenshot to know how it's working?

File: MaterialNavigationView/build.gradle [hunk 1]
```diff
@@ -6,13 +6,13 @@ apply plugin: "com.jfrog.bintray"
...
 android {
-    compileSdkVersion 29
+    compileSdkVersion 28
     buildToolsVersion "29.0.0"
 
 
     defaultConfig {
         minSdkVersion 19
-        targetSdkVersion 29
+        targetSdkVersion 28
```
Comment (PatilShreyas) [inline MaterialNavigationView/build.gradle]: Please don't change this file.

File: app/src/main/res/drawable/activated_background.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_pressed="true" android:drawable="@color/White" />
+    <item android:state_focused="true" android:drawable="@color/White" />
+    <item android:state_activated="true" android:drawable="@color/White" />
+    <item android:drawable="@android:color/transparent" />
```
Comment (PatilShreyas) [inline app/src/main/res/drawable/activated_background.xml]: Use a standard naming convention for `resources` and `attributes`.

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -18,11 +18,16 @@
...
+        android:fitsSystemWindows="true"
+        android:overScrollMode="never"
+        app:itemIconPadding="12dp"
         app:headerLayout="@layout/nav_header_main"
         app:insetForeground="@android:color/transparent"
+        app:itemBackground="@drawable/activated_background.xml"
         app:itemIconTint="@color/navigation_item_tint"
         app:itemStyle="default_style"
         app:itemTextColor="@color/navigation_item_tint"
+        app:theme="@style/NavigationDrawerStyle"
```
Comment (PatilShreyas) [inline app/src/main/res/layout/activity_main.xml]: This file shouldn't be changed.

File: app/src/main/res/values/styles.xml [hunk 4]
```diff
@@ -15,6 +15,12 @@
         <item name="windowNoTitle">true</item>
     </style>
 
+    <style name="NavigationDrawerStyle" parent="ThemeOverlay.AppCompat.navTheme">
+        <item name="android:textSize">14sp</item>
+        <item name="android:fontFamily">@font/robotomedium</item>
+        <item name="android:colorControlHighlight">@android:color/transparent</item><!--Here Removes Rectangle-->
+    </style>
+
```
Comment (PatilShreyas) [inline app/src/main/res/values/styles.xml]: For this, `styles.xml` shouldn't be involved. We have to do this in Library.

**Comment by zainfikrih** [comment]
@PatilShreyas 
Hi, I'm trying to solve this problem here #11

**Comment by PatilShreyas** [comment]
This issue is fixed and implemented in `v1.2`.

**Pull request closed without merging**